include/clad/Differentiator/VisitorBase.h

```diff
@@ -270,6 +272,13 @@ namespace clad {
     /// \returns Expression with correct Unary Operator placement.
     clang::Expr* ResolveUnaryMinus(clang::Expr* E, clang::SourceLocation OpLoc);
     clang::Expr* BuildParens(clang::Expr* E);
+    /// Sets Init as the initializer of the declaration VD and compute its
+    /// initialization kind. 
+    ///\param[in] VD - variable declaration 
+    ///\param[in] Init - can a nullptr, then only initialization kind is computed.
```

Comment:
```suggestion
    ///\param[in] Init - can be nullptr, then only initialization kind is
    ///                           computed.
```

This needs to be updated according to the comment on the implementation of the function.

Reply:
Done.